packages/ui/src/components/bundle-assets/__tests__/add-metric-report-asset-row-data.ts

```
@@ -83,7 +83,7 @@ describe('BundleAssets / addMetricReportAssetRowData', () => {
     });
   });
 
-  test('should add add data when curent run is null', () => {
+  test.only('should add data when current run is null', () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)